DQM/L1TMonitor/python/L1TStage2uGMT_cfi.py

```diff
@@ -117,3 +117,41 @@
     verbose = cms.untracked.bool(False),
 )
 
+# The five modules below compare the primary unpacked uGMT muon collection to goes to uGT board 0
```

Comment:
@davidlange6 so I should make one cfi file for every module and then group them in a cff?